project.clj

```diff
@@ -5,6 +5,7 @@
             :url "http://www.eclipse.org/legal/epl-v10.html"}
   :dependencies [[org.clojure/clojure "1.7.0"]
                  [org.clojure/clojurescript "1.7.170" :scope "provided"]
+                 [org.clojure/math.combinatorics "0.1.3" :scope "test"]
```

Comment:
If `:scope` works it is not the recommended way.  I think Leiningen started off with `:scope` and then changed to favoring profiles for this sort of thing.  If you are sure it is working, I guess you could leave it for now.  However, I'd say that'd worry me some across versions of lein.

There are two profiles automatically recognized by default for these sorts of dependencies, and they are `:dev` and `:provided`.  `:dev` is for "test" scope things.  When `lein pom` creates the Maven pom.xml it will be scoped as "test" as well.

The documentation for this is harder to find if you are searching for "scope" since lein doesn't have discussion on that terminology anymore.  The documentation is all on profiles.